lib/cucumber/formatter/publish_banner_printer.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'cucumber/term/banner'
+
+module Cucumber
+  module Formatter
+    class PublishBannerPrinter
+      include Term::Banner
+
+      def initialize(configuration)
+        configuration.on_event :test_run_finished do |_event|
+          display_banner(
+            [
+              [
+                'Share your Cucumber Report with your team at ',
+                ['https://reports.cucumber.io', :cyan, :bold]
+              ],
+              '',
+              [
+                'Command line option:    ',
+                ['--publish', :cyan]
+              ],
+              [
+                'Environment variable:   ',
+                ['CUCUMBER_PLUGIN_PUBLISH_ENABLED=true', :cyan]
+              ],
+              '',
+              [
+                'More information at ',
+                ['https://reports.cucumber.io/docs/cucumber-ruby', :cyan]
+              ],
+              '',
+              [
+                'To disable this message, specify ',
+                ['CUCUMBER_PUBLISH_QUIET=true', :dark],
```

Comment:
Change to `:bold` because `:dark` (grey??) isn't legible in black terminals. Fixed in Cucumber-JVM: https://github.com/cucumber/cucumber-jvm/commit/52c85472c78a52ac7174460f181e3f2e7f990b90

Reply:
done